### DISCUSSION TITLE
AI出錯?

### DISCUSSION THREAD

**Discussion by jerry850313** [1101-GS4719-C/Python-Community-GS4719-C#116] [閒聊八卦]
[英國交通相機誤將婦人當汽車！](https://www.dcfever.com/news/readnews.php?id=31680&fbclid=IwAR2LMS7T9DQlIuzTEfXaj2_qa1ORI8YxsK0EdKSA2ss0ZTqyg_st2I3JA7Q)
我覺得要探討這個問題之前，必須先了解這套系統的運作方式，我猜想這套系統可能是只認字不認車。如果再加一套判別是否為車的系統，應該能避免。此外，有些人會盜車牌或偷車來做壞事，如果再加一套判別是否為贓車(牌)，也許能避免犯罪。但是回到原點，為什麼只有判別文字的系統?我認為可能是警部系統沒有多餘的資源處理這些問題。

**Comment by chun1203** [comment]
我上學期修AI相關課程時，老師有教我們用龐大的資料庫讓機器學習，進而訓練出能判定是否為某樣物品的能力，所以目前的技術應該可以判別照到的物品是不是車子。

**Reply by jerry850313** [reply]
所以可以推測他們沒加上這項判斷

**Comment by 109302003** [comment]
Ai機器學習可以分辨出車子及人，會出錯可能ai資料量還不夠多

**Comment by cody910120** [comment]
就算是機器人也是會有犯錯的一天

**Comment by Isongzhe** [comment]
讓我想到之前特斯拉判斷不出維修車道，直接撞上維修車的新聞。感覺再怎麼避免都還是可能會發生類似的情況。

**Comment by 109802503gloria** [comment]
看了版主的推論覺得這樣的AI模型是不是有點太簡陋了，英國警部也太放心的讓它獨立運作了吧

**Comment by 107602505** [comment]
雖然這套系統是由採用AI的模型，還是要有人定期的監控跟維護，並且不斷更新他的一些小問題之類的才能運作得更長久也能讓人民更安心。

**Comment by YuHengChen54** [comment]
太恐怖了，要是辨識錯誤撞到人就出4了
